### PR TITLE
Minify SVG's

### PR DESCRIPTION
**Pull request by svenlombaert**
This PR resolves https://github.com/atomiclabs/cryptocurrency-icons/issues/210

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,58 +1,60 @@
...
-    "coinlist": "^3.0.0",
-    "husky": "^1.0.0-rc.13",
-    "xo": "^0.22.0"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run manifest"
-    }
-  }
+	"name": "cryptocurrency-icons",
```
Comment (sindresorhus) [inline package.json]: Don't change the indentation.

File: package.json [hunk 2]
```diff
@@ -1,58 +1,60 @@
...
+		"icon-pack",
+		"icon-set",
+		"erc20-tokens",
+		"erc20",
+		"erc721"
+	],
+	"scripts": {
+		"test": "xo",
+		"manifest": "node scripts/manifest.js",
+		"clean": "svgo -f svg/black && svgo -f svg/color && svgo -f svg/icon && svgo -f svg/white"
```
Comment (sindresorhus) [inline package.json]: Would be better to just glob all the SVG files.

File: package.json [hunk 3]
```diff
@@ -1,58 +1,60 @@
...
+	},
+	"devDependencies": {
+		"alpha-sort": "^2.0.1",
+		"coinlist": "^3.0.0",
+		"husky": "^1.0.0-rc.13",
+		"svgo": "^1.0.5",
```
Comment (sindresorhus) [inline package.json]: Remove the `^`. SVGO has a track-record of not following semver. We don't want SVGs to be ruined silently.

File: package.json [hunk 4]
```diff
@@ -1,58 +1,60 @@
...
+		"xo": "^0.22.0"
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "npm run manifest"
```
Comment (sindresorhus) [inline package.json]: Shouldn't `clean` be there?

File: svg/black/$pac.svg [hunk 5]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg width="32" height="32" xmlns="http://www.w3.org/2000/svg"><path d="M16 32C7.163 32 0 24.837 0 16S7.163 0 16 0s16 7.163 16 16-7.163 16-16 16zm-.065-28C9.342 4.001 4 9.347 4 15.94c.001 6.592 5.347 11.936 11.94 11.935 6.592-.002 11.936-5.347 11.935-11.94A11.937 11.937 0 0 0 15.935 4zm-4.847 5.23a.123.123 0 0 1 .125-.12h4.973a6.98 6.98 0 0 1 3.027.628c.18.088.357.186.528.292l-3.237 1.853h-1.689a.123.123 0 0 0-.12.126v.932l-3.607 2.086V9.23zM9.749 19.708l-2.162-2.184 7.12-4.108v2.662a.123.123 0 0 0 .125.12h1.017l-6.093 3.51h-.007zm10.662-2.03c-1.044.872-2.477 1.308-4.299 1.308h-1.297a.123.123 0 0 0-.123.123v2.688l-1.81 1.042-1.69.977-.107.06v-4.455l5.603-3.224a2.056 2.056 0 0 0 1.297-.513 1.74 1.74 0 0 0 .318-.432l3.578-2.071c.06.328.09.66.089.994.004 1.463-.516 2.63-1.56 3.503zm-1.92-3.019c.024-.153.036-.309.037-.464a2.417 2.417 0 0 0-.563-1.674A1.823 1.823 0 0 0 17.15 12l4.973-2.871 2.182 2.183-5.814 3.347z" fill="#000" fill-rule="nonzero"/></svg>
```
Comment (sindresorhus) [inline svg/black/$pac.svg]: `viewBox` should not be removed.

File: package.json (hunk 4 above)
Comment (svenlombaert) [reply package.json]: Even if you run it on pre-commit, it won't trigger an error?

File: svg/black/$pac.svg (hunk 5 above)
Comment (sindresorhus) [reply svg/black/$pac.svg]: `viewBox` ensures aspect ratio.

**Pull request merged**